### PR TITLE
feat(metrics): populate metric values for content server backend Glean

### PR DESCRIPTION
**Pull request by chenba**
Because:
 - the content server backend Glean pings did not have actual values for most of their metrics

This commit:
 - populates the "common" metrics in the ping
   - the approach is currently based on the /metrics-flow request; we can evolve it as necessary as we support more events
 - combines the node-uap based useragent lib in auth server into the one in fxa-shared, since the content server backend need the "device type"

### CODE REVIEW

**Comment by chenba** [comment]
@vbudhram do you mind reviewing this, since you reviewed part I?  Thanks!

File: packages/fxa-auth-server/package.json [hunk 1]
```diff
@@ -113,7 +113,6 @@
     "moment-timezone": "^0.5.35",
     "mozlog": "^3.0.2",
     "mysql": "^2.18.1",
-    "node-uap": "https://github.com/mozilla-fxa/node-uap.git#494714db3e65a4dbfae772f536c8697269174574",
```
Comment (vbudhram) [inline packages/fxa-auth-server/package.json:116]: Do you know why this was pinned?

**Review by vbudhram — APPROVED** [review]
@chenba Code looks good, I was worried that we were not using the same logic to parse the user agent but looks like fxa-shared has the same pinned version. Thanks!

File: packages/fxa-auth-server/package.json (hunk 1 above)
Comment (chenba) [reply packages/fxa-auth-server/package.json:116]: I don't know the whole history.  It was pinned to Danny's fork.  Then Dan's.  Then it moved to mozilla-fxa.  node-uap also deleted some commits on the original repo so maybe that was a motivation to keep a fork?

**Pull request merged**